locust/argument_parser.py

```diff
@@ -271,14 +252,27 @@ def wait_for_reply():
         sys.stderr.write(f"Got error from master: {msg.data['error']}\n")
         sys.exit(1)
 
-    filename = msg.data["filename"]
-    with open(os.path.join(tempfile.gettempdir(), filename), "w", encoding="utf-8") as locustfile:
-        locustfile.write(msg.data["contents"])
+    tempclient.close()
+    return msg.data
 
-    atexit.register(exit_handler, locustfile.name)
```

Comment:
Was this accidentally deleted?

Reply:
No it's not needed anymore, we no longer open any files here so this `atexist` is no longer necessary. Instead we use re-use the same `parse_locustfile_paths` that is used in Master. And that function handles everything with reading the file and registering the `atexit`.